yw/Software/src/spellchecker/ReadTest.java

```diff
@@ -0,0 +1,26 @@
+package spellchecker;
```

Comment:
Should this variable be in test package?

Reply:
The reason of importing spellchecker is to use the function in checker 2, I think it is reasonable to import spellchecker

Reply:
Sorry, I mistyped: 
Should this class (ReadTest) be in test package?